cmplibrary/src/main/res/raw/js_receiver.js

```diff
@@ -24,7 +24,7 @@ function dataFromMessage(msgEvent) {
     return {
         name: msgEvent.data.name,
         actionType: msgEvent.data.actions.length ? msgEvent.data.actions[0].data.type : null,
-        choiceId: msgEvent.data.actions.length ? msgEvent.data.actions[0].data.choice_id : null,
```

Comment:
Should we add one unit test for the case in which the `choiceId` is `Number` or `String`? Specially now that we're jesting the shit out of our JSReceiver